src/jailer/src/env.rs

```diff
@@ -355,6 +383,19 @@ impl Env {
         write!(pid_file, "{}", pid).map_err(|err| JailerError::Write(pid_file_path, err))
     }
 
+    fn get_userfaultfd_minor_dev_number() -> Result<u32, UserfaultfdParseError> {
+        let buf = read_to_string("/proc/misc")?;
+
+        for line in buf.lines() {
+            let dev: Vec<&str> = line.split(' ').collect();
+            if dev[1] == "userfaultfd" {
```

Comment:
I know, not likely, but is the `/proc/misc` file format guaranteed? Is it possible that `dev[1]` doesn't exist?

Reply:
It will be there. If there is a line it will refer to the minor node (dev[0]) and device name (dev[1]).

Reply:
However, Patrick had the same observation and it doesn't hurt checking, so I'm adding the check.